src/app/address/googleAutocomplete/GoogleAutocompleteFormField.tsx

```diff
@@ -40,10 +40,13 @@ const GoogleAutocompleteFormField: FunctionComponent<GoogleAutocompleteFormField
         <TranslatedString id="address.address_line_1_label" />
     ), []);
 
+    const labelID = getAddressFormFieldLabelId(name);
```

Comment:
```suggestion
    const labelId = getAddressFormFieldLabelId(name);
```

Reply:
Thanks for the suggestion. I have replaced four occurrences of `labelID` with `labelId` in a new commit.